README.md

```diff
@@ -12,7 +12,7 @@ to your `devcontainer.json`:
 "runArgs": ["--device=/dev/net/tun"],
 "features": {
   // ...
-  "ghcr.io/tailscale/codespace": {}
+  "ghcr.io/tailscale/codespace/tailscale:1": {}
```

Comment:
No, this is what we've been working through with:
- https://github.com/devcontainers/devcontainers.github.io/pull/127
- https://github.com/devcontainers/devcontainers.github.io/pull/119
- https://github.com/devcontainers/action/issues/147

The namespace has to be `ghcr.io/tailscale/codespace` for GitHub to list it. They rejected `ghcr.io/tailscale/codespace/tailscale`

Reply:
Right, `ghcr.io/tailscale/codespace` is the namespace but `ghcr.io/tailscale/codespace/tailscale` is required to make it work in the features list. It's the same at https://github.com/devcontainers/devcontainers.github.io/pull/119/files#diff-213a25c23998b235e92e81da59bbf14a591fa6332ee3d14a662ad4471e621bcbR230

Reply:
Yep, as written in the README doesn't work, but with this change the codespace can be created successfully!

In the docs (https://tailscale.com/kb/1160/github-codespaces/) explain that you should set up your codespace as

```
    "runArgs": ["--device=/dev/net/tun"],
    "features": {
        // ...
        "ghcr.io/tailscale/codespace/tailscale": {}
        // ...
    }
```

– this README and those docs should agree!